nodes.go

```diff
@@ -95,14 +95,20 @@ func (n *Node) Containers(ctx context.Context) (c Containers, err error) {
 }
 
 func (n *Node) Container(ctx context.Context, vmid int) (*Container, error) {
-	var c Container
+	c := &Container{
+		client: n.client,
+		Node:   n.Name,
+	}
+
 	if err := n.client.Get(ctx, fmt.Sprintf("/nodes/%s/lxc/%d/status/current", n.Name, vmid), &c); err != nil {
```

Comment:
pretty sure if you make `c` a pointer already you don't have to use an `&` here and probably not on line 107 for the container config since it too is already a pointer. you probably want to double check me on that and make sure the return is populated but it passed build and lint for me without it.

Reply:
I thought the same thing, but the value of `c` is `nil` when I don't use the `&` here.